test/create-adapter.test.ts

```diff
@@ -249,15 +249,16 @@ describe("adapter creation =>", () => {
 					...baseAnswers,
 					language: "JavaScript",
 					es6class: "yes",
-					adminReact: "yes",
+					adminUi: "react",
 				};
 				await expectSuccess("adapter_JS_React", answers);
 			});
 
-			it("Adapter, JavaScript, ESLint, Spaces, Single quotes, Apache License", async () => {
+			it("Adapter, JavaScript, JSON UI, ESLint, Spaces, Single quotes, Apache License", async () => {
```

Comment:
I'd mention the JSON UI in the test name so it appears in the directory name, line 268